python/lsst/sconsUtils/installation.py

```diff
@@ -95,6 +95,13 @@ def setPrefix(env, versionString, eupsProductPath=None):
                 "%s\nFound problem with version number; update or specify force=True to proceed"
                 % err
             )
+
+    if state.env['no_eups']:
+        if 'prefix' in env and env['prefix']:
+            return env['prefix']
+        else:
+            return "/usr/local"
+
```

Comment:
Oh, I don't know; I know virtually nothing about OSX.  I just wanted to make sure this wasn't a Linux-centric assumption.